x/bridge/keeper/keeper.go

```diff
@@ -373,6 +379,23 @@ func (k Keeper) CalculateValidatorSetCheckpoint(
 	return checkpoint, nil
 }
 
+// check whether last saved validator set is too old (more than 2 weeks)
+func (k Keeper) LastSavedValidatorSetStale(ctx context.Context) (bool, error) {
```

Comment:
would it be possible that we don't update the validator set in 2 weeks?

Reply:
it's possible, if the validator set just doesn't change by at least 5% during that 2 week period. We need this check just to make sure the bridge doesn't get frozen. If the validator set in the bridge contract is >3 weeks old, no more bridge proofs can be run until the team multisig resets the validator set in the contract. We don't really want to have to use this multisig backstop.